src/libraries/System.Web.HttpUtility/tests/HttpUtility/HttpUtilityTest.cs

```diff
@@ -443,6 +443,16 @@ public void ParseQueryString_ToString()
             Assert.Equal(expected, HttpUtility.ParseQueryString(expected).ToString());
         }
 
+
+        [Fact]
+        public void ParseQueryString_nullValue_ToString()
+        {
+            var nameValueCollection = System.Web.HttpUtility.ParseQueryString(string.Empty);
+            nameValueCollection.Add("baz", null);
+            var s = nameValueCollection.ToString();
```

Comment:
Normally I'd say we shouldn't use var here, but you're just being consistent with what's already in the file, so it's fine.